python/rikai/parquet/dataset.py

```diff
@@ -228,3 +225,50 @@ def __iter__(self):
             yield self._convert(
                 shuffler.pop().to_dict(), self.spark_row_metadata
             )
+
+    def to_pandas(self):
```

Comment:
is this user-facing API?  lets add a comment to mention that this method loads all data into memory?

Reply:
I'll add some docstrings.
I was thinking I can add a `limit` parameter that's just pq.ParquetDataset.head (https://arrow.apache.org/docs/python/generated/pyarrow.dataset.Dataset.html#pyarrow.dataset.Dataset.head) if you only want a preview